rule/rule.go

```diff
@@ -996,7 +996,7 @@ func (r *Rule) sync() {
 func ShouldKeep(e bzl.Expr) bool {
 	for _, c := range append(e.Comment().Before, e.Comment().Suffix...) {
 		text := strings.TrimSpace(strings.TrimPrefix(c.Token, "#"))
-		if text == "keep" {
+		if strings.HasPrefix(text, "keep") {
```

Comment:
This would also match `# keeping to mess this up` or `# keep this around for the time being` which may be unintentional. 

How about this?

```suggestion
		if text == "keep" || strings.HasPrefix(text, "keep: ") {
```